libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -366,6 +366,7 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             printf("Creating model %s at speed %.1f\n", model_str, speedup);
             sitl_model = model_constructors[i].constructor(home_str, model_str);
             sitl_model->set_interface_ports(simulator_address, simulator_port_in, simulator_port_out);
+            sitl_model->set_rate_hz (_framerate);
```

Comment:
**_framerate**  is already exists and I believe this was the reason of it.
do you suggest that we should remove this variable instead of trainfering its value to backend?